plugins/generator-1.20.1/forge-1.20.1/templates/particle.java.ftl

```diff
@@ -64,7 +64,7 @@ package ${package}.client.particle;
 		this.spriteSet = spriteSet;
 
 		this.setSize(${data.width}f, ${data.height}f);
-		<#if data.scale != 1>this.quadSize *= ${data.scale}f;</#if>
+		<#if !hasProcedure(data.scale)>this.quadSize *= ${data.scale}f;</#if>

```

Comment:
You are now ignoring case where scale is 1 and you don't need this line of code. Please keep existing logic in place too. Applies to both versions

Reply:
it gives me an error if I leave the old test (<#if data.scale != 1>)

Reply:
You need to check for the fixed value, not the procedure object.

Also if you don't know something, you should ask, not silently remove features..

Reply:
Fixed